package.json

```diff
@@ -1,64 +1,71 @@
 {
-  "title": "Tus.io",
   "name": "tus-io",
-  "private": true,
-  "version": "0.0.1",
+  "version": "1.0.0",
+  "main": "astro.config.mjs",
+  "packageManager": "yarn@3.5.0",
   "scripts": {
-    "build:assets:production": "LANYON_ENV=production LANYON_JEKYLL=${PWD}/_jekyll/jekyll.sh lanyon build:assets",
-    "build:production": "LANYON_ENV=production LANYON_JEKYLL=${PWD}/_jekyll/jekyll.sh lanyon build",
-    "build": "LANYON_JEKYLL=${PWD}/_jekyll/jekyll.sh lanyon build",
-    "deploy": "LANYON_JEKYLL=${PWD}/_jekyll/jekyll.sh lanyon deploy",
-    "download:logos": "cross-env DEBUG=*:* IMAGES_CFG=_data/logos.yml IMAGES_DIR=assets/img/logos node ./_scripts/download-external-images.js",
-    "download:press": "cross-env DEBUG=*:* IMAGES_CFG=_data/press.yml IMAGES_DIR=assets/img/press node ./_scripts/download-external-images.js",
-    "fix": "eslint . --fix && yarn fix:formatting",
-    "inject:community": "_scripts/inject-community.sh",
-    "inject:protocol": "_scripts/inject-protocol.sh",
-    "inject": "npm-run-all --parallel inject:*",
-    "lint": "eslint . && yarn lint:formatting",
-    "optimize:logos": "cross-env IMAGES_CFG=_data/logos.yml IMAGES_DIR=assets/img/logos _scripts/images-optimize.sh",
-    "optimize:press": "cross-env IMAGES_CFG=_data/press.yml IMAGES_DIR=assets/img/press _scripts/images-optimize.sh",
-    "pull": "git pull",
-    "serve:production": "LANYON_ENV=production LANYON_JEKYLL=${PWD}/_jekyll/jekyll.sh lanyon serve",
-    "serve": "LANYON_JEKYLL=${PWD}/_jekyll/jekyll.sh lanyon serve",
-    "start:production": "npm run build:production && npm run serve:production",
-    "start": "LANYON_JEKYLL=${PWD}/_jekyll/jekyll.sh lanyon start",
-    "test": "true",
-    "upgrade:modules": "updtr",
-    "fix:formatting": "prettier --write .",
-    "lint:formatting": "prettier --check ."
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "lint": "concurrently \"npm:lint:*\"",
+    "lint:js": "eslint .",
+    "lint:ts": "tsc",
+    "lint:fmt": "prettier --check .",
+    "inject": "concurrently \"npm:inject:*\"",
+    "inject:protocol": "sh scripts/inject-protocol.sh"
```

Comment:
best make the script executable and let it determine what the interpreter should be